changelog/changes/2023-10-17-update-engine-torcx-removal.md

```diff
@@ -0,0 +1 @@
+- update_engine: remove torcx sanity checks because the OS image removed torcx ([update_engine#30](https://github.com/flatcar/update_engine/pull/30))
```

Comment:
Same here, this is not interesting for the user. What I actually would want to read as a user is that torcx is gone and what to do now - a pointer to https://www.flatcar.org/docs/latest/provisioning/sysext/ should be enough for the details but a good one sentence summary should tell that torcx is removed because systemd-sysext provides the same functionality in a more generic and distro-independent way.

Reply:
That's a good point. Will add that and the docs pointer, and mention the relevant changes for OS, update_engine and mantle there.